introduction.md

```diff
@@ -1,2 +1,8 @@
 Hi! My name is Antonio. Nice to meet you Sensei.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?